src/crepe/types.h

```diff
@@ -27,4 +27,17 @@ typedef Vector2<float> vec2;
 //! Default Vector2<double> type
 typedef Vector2<double> dvec2;
 
+class Sprite;
+class Camera;
+
+struct RenderCtx{
```

Comment:
Deze struct is specifiek voor SDLcontext kun je hem daar neerzetten?
ook in de klasse zelf